deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -14,34 +14,40 @@ spec:
       labels:
         app: elastic-agent
     spec:
+      # To run Elastic Agent on Kubernetes master nodes
+      # Agents running on master nodes collect metrics from the control plane components (scheduler, controller manager) of Kubernetes
       tolerations:
         - key: node-role.kubernetes.io/master
           effect: NoSchedule
       serviceAccountName: elastic-agent
       hostNetwork: true
+      # 'hostPID: true' enables the Elastic Security integration to observe all process exec events on the host.
       # Sharing the host process ID namespace gives visibility of all processes running on the same host.
-      # This enables the Elastic Security integration to observe all process exec events on the host.
       hostPID: true
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: elastic-agent
           image: docker.elastic.co/beats/elastic-agent:8.3.0
           env:
+            # Set to 1 for enrollment into Fleet server. If not set, Elastic Agent is run in standalone mode
             - name: FLEET_ENROLL
               value: "1"
-            # Set to true in case of insecure or unverified HTTP
+            # Set to true to communicate with Fleet with either insecure HTTP or unverified HTTPS
             - name: FLEET_INSECURE
               value: "true"
-              # The ip:port pair of fleet server
+            # URL to enroll the Fleet Server into
             - name: FLEET_URL
               value: "https://fleet-server:8220"
-              # If left empty KIBANA_HOST, KIBANA_FLEET_USERNAME, KIBANA_FLEET_PASSWORD are needed
+            # The token to use for Fleet enrollment
+            # If left empty KIBANA_HOST, KIBANA_FLEET_USERNAME, KIBANA_FLEET_PASSWORD are needed
             - name: FLEET_ENROLLMENT_TOKEN
               value: ""
             - name: KIBANA_HOST
               value: "http://kibana:5601"
+            # The basic authentication username used to connect to Kibana and retrieve a service_token to enable Fleet
             - name: KIBANA_FLEET_USERNAME
```

Comment:
Do we need a link to direct users where to find those values? 
Afterall this is the comment for, if dont know where to search

Reply:
In managed yes. Maybe was not the best place to put the comment.

But you get my point, that all of this values are somewhere in the portal and user needs point of reference. If makes sense.

Reply:
I've added info for FLEET_URL and enrollment token - https://github.com/elastic/elastic-agent/pull/708/commits/d9c4344745253ba4fdeb3071cdd51741f8012754. Regarding KIBANA_*: it depends on the setup - if elastic cloud or self-managed and I couldn't find generic documentation